deploy/csi-hostpath-plugin.yaml

```diff
@@ -56,7 +56,7 @@ spec:
               name: socket-dir
             - mountPath: /registration
               name: registration-dir
-            - mountPath: /csi-data-dir
+            - mountPath: /run/csi-data-dir
```

Comment:
Jan's comment was specifically /run/csi driver name,

i.e. https://github.com/openshift/csi-driver-shared-resource/blob/master/vendor/github.com/openshift/api/operator/v1/types_csi_cluster_driver.go#L55

of course you cannot use that constant in yaml, but use the string that constant is set to